juju/juju-qa-jenkins
# Add another linter which asserts multijobs are alphabetical

Previously this rule existed informally and wasn't enforced, leading to much frustration re-sorting them every now and then. So add this as a linting rule

### QA Steps

- `make static-analysis` should pass
- re-order a couple of jobs in some multijob. `make static-analysis` should now fail
